xbmc/guilib/StereoscopicsManager.cpp

```diff
@@ -46,6 +46,7 @@
 #include "utils/Variant.h"
 #include "rendering/RenderSystem.h"
 #include "guiinfo/GUIInfoLabels.h"
+#include "cores/DataCacheCore.h"
```

Comment:
Is this header actually needed since we use the serviceBroker to get the object? Been a while since I did some C++

Reply:
Yes, because `GetDataCacheCore()` returns a forward-declared reference to CDataCacheCore, but we need to include the header if we use it or we'll get a "Use of undefined class" error.